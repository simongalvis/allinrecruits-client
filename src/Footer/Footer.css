.Footer{
    clear: both;
    bottom: 0;
    background-color: black;
    position: relative;
    width:100%;
    color:white; 
}
.portfolio-link{
    color:white;
    font-size:13px;
}