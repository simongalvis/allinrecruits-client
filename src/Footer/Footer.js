import React from 'react'
import './Footer.css'

class Footer extends React.Component{
    
render(){
    return(
        <div className="Footer">
            <h3>Footer</h3>
        </div>
    )
}
}

export default Footer;