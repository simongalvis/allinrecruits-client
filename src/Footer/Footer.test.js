import React from 'react'
import ReactDOM from 'react-dom'
import Footer from './Footer'
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom'

describe(`Footer component`, () =>{
    it(`renders without crashing`, () =>{
        const div = document.createElement('div');

        ReactDOM.render(<BrowserRouter><Footer/></BrowserRouter>, div)

        ReactDOM.unmountComponentAtNode(div);
    })
    it(`Renders UI as expected`, () =>{
        const tree = renderer
            .create(<BrowserRouter><Footer/></BrowserRouter>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})