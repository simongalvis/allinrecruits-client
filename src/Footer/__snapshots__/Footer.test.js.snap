// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Renders UI as expected 1`] = `
<div
  className="Footer"
>
  <p
    className="portfolio-link"
  >
    Â© - 
    <a
      className="portfolio-link"
      href="http://simongalvis.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      simongalvis.com
    </a>
  </p>
</div>
`;
